nucliadb_models/tests/test_search.py

```diff
@@ -30,40 +30,40 @@ def test_filter_model_validator():
 def test_field_extension_strategy_fields_field_validator():
     search.FieldExtensionStrategy(
         name="field_extension",
-        fields={"f/myfield"},
+        fields=["f/myfield"],
     )
 
     # not a set of fields
     with pytest.raises(ValidationError):
         search.FieldExtensionStrategy(
             name="field_extension",
-            fields=0,
+            fields=0,  # type: ignore
         )
 
     # fields must be in the format {field_type}/{field_name}
     with pytest.raises(ValidationError):
         search.FieldExtensionStrategy(
             name="field_extension",
-            fields={"myfield"},
+            fields=["myfield"],
         )
 
     # not an allowed field
     with pytest.raises(ValidationError):
         search.FieldExtensionStrategy(
             name="field_extension",
-            fields={"z/myfield"},
+            fields=["z/myfield"],
         )
 
 
 def test_find_request_fulltext_feature_not_allowed():
     with pytest.raises(ValidationError):
-        search.FindRequest(features=[search.SearchOptions.FULLTEXT])
+        search.FindRequest(features=[search.SearchOptions.FULLTEXT])  # type: ignore
 
 
 def test_find_supports_search_options():
-    search.FindRequest(features=[search.SearchOptions.KEYWORD])
-    search.FindRequest(features=[search.SearchOptions.SEMANTIC])
-    search.FindRequest(features=[search.SearchOptions.RELATIONS])
+    search.FindRequest(features=[search.FindOptions.KEYWORD])
+    search.FindRequest(features=[search.FindOptions.SEMANTIC])
+    search.FindRequest(features=[search.FindOptions.RELATIONS])
```

Comment:
This isn't a mistake. It is testing that you can actually pass `SearchOptions` to `FindRequest` as this was a legacy behavior we should support (at least for a while to not break users)